sass/_variables.scss

```diff
@@ -0,0 +1,16 @@
+//variables are defined here
+
+//font size placeholders
+%tiny-font {
+	font-size: $tiny-font-size;
```

Comment:
Your comments don't align with reality here -- `$tiny-font-size` is defined in your _fonts.scss file.

I would've merged this file and mixins (renamed, perhaps, `_utilities.scss`), since you don't have a ton of either.